pom.xml

```
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>by.itacademy.javaenterprise.borisevich</groupId>
+    <artifactId>lecture-17</artifactId>
+    <packaging>jar</packaging>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>lecture-17</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <mysql-connector-java-version>5.1.49</mysql-connector-java-version>
+        <slf4jVersion>1.7.30</slf4jVersion>
```

```suggestion
        <slf4j.version>1.7.30</slf4j.version>
```
camelCase в maven не используется. в следующих задания будет -1